### PR TITLE
Add code for LEAD_02, LEAD_03, LEAD_04

### PR DESCRIPTION
**Pull request by vtarasiuk**

### CODE REVIEW

File: docs/use cases/README.md [hunk 1]
```diff
@@ -1,149 +1,172 @@
...
-    
-    right footer
-        Аналітичний портал. Модель прецедентів.
-        НТУУ КПІ ім.І.Сікорського
-        Киів-2020
-    end footer
+    |Користувач|
+        start
+        :Користувач авторизується
+        в системі;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Remove these lines.

File: docs/use cases/README.md [hunk 2]
```diff
@@ -1,149 +1,172 @@
...
+
+        :Користувач відкриває вікно
+        з усіма завданнями;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Would be better kinda:
1. User clicks the button "Tasks"
2. The system is sending task list to user

...

File: docs/use cases/README.md [hunk 3]
```diff
@@ -1,149 +1,172 @@
...
+
+        :Користувач обирає потрібне
+        завдання;
+
+        :Користувач натискає
+        кнопку "Змінити дедлайн";
+        
+        :Користувач обирає дедлайн
+        для завдання;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Would be better kinda:
1. The system is sending deadline calendar to user
2. User chooses a deadline for the task

...

File: docs/use cases/README.md [hunk 4]
```diff
@@ -1,149 +1,172 @@
...
+
+        <font color=000 size=16><b>ОСНОВНИЙ СЦЕНАРІЙ:</b>
+
+    end header
 
-</center>
+    |Користувач|
+        start
+        :Користувач авторизується
+        в системі;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Remove these lines.

File: docs/use cases/README.md [hunk 5]
```diff
@@ -1,149 +1,172 @@
...
+
+        :Користувач відкриває вікно
+        з усіма завданнями;
+
+        :Користувач обирає потрібне
+        завдання;
+
+        :Користувач ознайомлюється 
+        з списком розробників;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Remove these lines.

File: docs/use cases/README.md [hunk 6]
```diff
@@ -1,149 +1,172 @@
...
-    
-    right footer
-        Аналітичний портал. Модель прецедентів.
-        НТУУ КПІ ім.І.Сікорського
-        Киів-2020
-    end footer
+    |Користувач|
+        start
+        :Користувач авторизується
+        в системі;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Remove these lines.

File: docs/use cases/README.md [hunk 7]
```diff
@@ -1,149 +1,172 @@
...
+    end header
 
-</center>
+    |Користувач|
+        start
+        :Користувач авторизується
+        в системі;
+
+        :Користувач відкриває вікно
+        з усіма завданнями;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Would be better kinda:

1. User clicks the button "Tasks"
2. The system is sending task list to user

...

File: docs/use cases/README.md [hunk 8]
```diff
@@ -1,149 +1,172 @@
...
-        Киів-2020
-    end footer
+    |Користувач|
+        start
+        :Користувач авторизується
+        в системі;
 
-@enduml
+        :Користувач відкриває вікно
+        з усіма завданнями;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Would be better kinda:

1. User clicks the button "Tasks"
2. The system is sending task list to user

...

File: docs/use cases/README.md [hunk 9]
```diff
@@ -1,149 +1,172 @@
...
 
-**Діаграма прецедентів**
+        :Користувач обирає потрібне
+        завдання;
+
+        :Користувач перевіряє 
+        завдання;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Remove these lines.

File: docs/use cases/README.md [hunk 10]
```diff
@@ -1,149 +1,172 @@
...
+        
+        :Користувач вводить
+        результат перевірки;
+        
+        :Користувач зберігає
+        введену інформацію;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Remove these lines.

File: docs/use cases/README.md [hunk 11]
```diff
@@ -1,149 +1,172 @@
...
 
-**Діаграма прецедентів**
+        :Користувач обирає потрібне
+        завдання;
+
+        :Користувач перевіряє 
+        завдання;
+        
+        :Користувач вводить
+        результат перевірки;
```
Comment (gurug-prog) [inline docs/use cases/README.md]: Replace word "вводить" with "заносить".

**Review by gurug-prog — CHANGES_REQUESTED** [review]
Good! But there are some common mistakes you have to correct.

**Review by gurug-prog — APPROVED** [review]
LGTM!

**Review by socheretenko — COMMENTED** [review]
LGTM

**Pull request merged**